src/bindings/python/tests/test_transformations/test_pattern_ops.py

```diff
@@ -85,6 +86,104 @@ def test_any_input_predicate():
     assert not matcher.match(slope)
 
 
+@pytest.mark.skip("Optional is not working properly yet CVS_133523")
+def test_optional_full_match():
```

Comment:
Do we have any tests that are not skipped?
I believe that some of them should pass.